docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     container_name: luigi
     volumes:
       # Code
-      - ./:/luigi/tasks/qgreenland:ro
```

Comment:
I'd prefer to only make a portion of the code dir `rw`; what do you think of having a `config/helpers/ancillary` dir that is mounted `rw` and keeping the rest of the code `rw`?